smart-contracts/osmosis/contracts/cl-vault/tests/test-tube/autocompound.rs

```diff
@@ -366,24 +383,19 @@ fn test_autocompound_with_rewards_swap_non_vault_funds() {
     )
     .unwrap();
 
-    // Assert balances after autocompound of funds
-    let (expected_refund_base, expected_refund_quote) =
-        calculate_expected_refunds(49500000000, 49500000000, deposit_ratio);
-    // Base about rewards swapped to token base
-    let balances_after_autocompound_base =
+    let after_autocompound_base_balance =
         get_balance_amount(&app, contract_address.to_string(), DENOM_BASE.to_string());
-    assert_approx_eq!(
-        expected_refund_base,
-        balances_after_autocompound_base,
-        &deposit_ratio_approx
-    );
-    // Quote about rewards swapped to token quote
-    let balances_after_autocompound_quote =
+    let after_autocompound_quote_balance =
         get_balance_amount(&app, contract_address.to_string(), DENOM_QUOTE.to_string());
-    assert_eq!(expected_refund_quote, balances_after_autocompound_quote);
+    // Assert that either the base balance is 0, or the quote balance is 0, or both are 0.
```

Comment:
that is easy to understand from the following line. though the comment is confusing as it explicitly mentions the case that both are 0, which is already handled and thus not checked for in the implementation